packages/@uppy/status-bar/src/Components.js

```diff
@@ -0,0 +1,443 @@
+const classNames = require('classnames')
+const throttle = require('lodash.throttle')
+const prettierBytes = require('@transloadit/prettier-bytes')
+const prettyETA = require('@uppy/utils/lib/prettyETA')
+const { h } = require('preact')
+
+const statusBarStates = require('./StatusBarStates')
+
+const renderDot = () => ' \u00B7 '
```

Comment:
```suggestion
const DOT = `\u00B7`
const renderDot = () => ` ${DOT} `
```

But why is it a function?

```suggestion
const DOT = `\u00B7`
const SPACED_DOT = ` ${DOT} `
```